neqo-http3/src/connection.rs

```diff
@@ -534,7 +534,9 @@ impl Http3Connection {
             .send_streams
             .get_mut(&stream_id)
             .ok_or(Error::InvalidStreamId)?;
-        send_stream.close(conn)?;
+        // The following funcion may return InvalidStreamId from the transport layer if the stream has been cloesd
+        // already. It is ok to ignore it here.
```

Comment:
Maybe we might put here an assertion that the error is InvalidStreamId for case that transport layer is changed and can throw other errors as well. But I'll leave it up to you...

Reply:
This is a good idea. I will add this assertion in a follow up.